tuxemon/resources/db/locale/eo.json

```diff
@@ -5,18 +5,18 @@
     "combat_used_x_on_y": "${{name}} estis uzata al ${{target}} de ${{user}}!",
     "combat_call_tuxemon": "Iru, ${{name}}!",
     "combat_opponent_call_tuxemon": "${{user}} sendas ${{name}}!",
-    "combat_state_poison_damage": "${{name}} estis venenigita!",
-    "combat_status_damage": "${{name}} estis damaĝita de ${{status}}!",
+    "combat_state_poison_damage": "${{name}} veneniĝis!",
+    "combat_status_damage": "${{name}} damaĝiĝis pro ${{status}}!",
     "combat_victory": "Vi gajnis!",
     "combat_defeat": "Vi estis venkita!",
-    "combat_capturing success": "Vi kaptis ${{name}}!",
+    "combat_capturing success": "${{name}} estias kaptita!",
     "combat_capturing_fail": "${{name}} liberiĝis!",
     "combat_replacement": "Elektu anstataŭanton!",
     "combat_replacement_is_fainted": "La monstro svenis!",
     "combat_wild_appeared": "Sovaĝa ${{name}} aperis!",
-    "combat_monster_choice": "Kion ${{name}} faris?",
+    "combat_monster_choice": "Kion ${{name}} faru?",
     "combat_fainted": "${{name}} svenas!",
```

Comment:
Same comment about tense, the English string uses past, this string is present. This could be a good translation depending on context. Just making sure this isn't accidental.